### PR TITLE
medium modals should have 32 px above actions

### PR DESCRIPTION
**Pull request by mayakoneval**
https://app.zeplin.io/project/5c7dcb5ab4e654bca8cde54d/screen/5d0ad14c42474a710911cccd

### CODE REVIEW

File: src/Modal/Modal.tsx [hunk 1]
```diff
@@ -168,7 +168,7 @@ export function Modal({
               alignItems: "baseline",
               display: "flex",
               justifyContent: "flex-end",
-              marginTop: 24,
+              marginTop: size === "medium" ? 32 : 24,
```
Comment (evans) [inline src/Modal/Modal.tsx]: What about large?

Comment (mayakoneval) [reply src/Modal/Modal.tsx]: large also has 24px ^ zeplin shows!

**Comment by mayakoneval** [comment]
im not authorized to merge this so @justinanastos i imagine will merge?

**Comment by evans** [comment]
@mayakoneval It looks like large is 40px and small is 32px from the zeplin. Do you see where the 24px comes from?

**Comment by mayakoneval** [comment]
@evans im referring to this spacing here

image

**Comment by justinanastos** [comment]
:rocket: PR was released in v2.0.2 :rocket:

**Pull request merged**